cookbooks/cdo-varnish/libraries/http_cache.rb

```diff
@@ -232,6 +232,11 @@ def self.config(env)
             headers: WHITELISTED_HEADERS + ALLOWED_WEB_REQUEST_HEADERS,
             cookies: whitelisted_cookies
           },
+          {
+            path: 'tracking_pixel',
```

Comment:
I _think_ this should have a leading slash.
```suggestion
            path: '/tracking_pixel',
```